### DISCUSSION TITLE
Enable algorithm option for trending

### DISCUSSION THREAD

**Discussion by Sithic** [robertodoering/harpy#661] [Ideas]
Feature Request

I noticed that Trending in harpy compared to Twitter app was lacking all of the things i was actually interested in, going from Sykkuno and gaming news to Madrid and Messi spam 

Hoping to see an option to enable the personalized trending stuff.

**Comment by robertodoering** [comment]
Twitter only allows you to retrieve the home timeline in (reverse) chronological order and not in their algorithmic way.
You can use the filter to hide certain tweets from your timeline (e.g. retweets or tweets that include a hashtag that you don't like).

**Reply by Sithic** [reply]
i meant this in the sense of like the Trending topics aren't related to users interests but rather overall most posts in a location of choice, but never interest related. I'm not sure if that is possible to implement, unless Twitter goes Open Source maybe

**Reply by robertodoering** [reply]
I see, same scenario with the Trends though, we don't get results tailored for the user - we just get all trends based on a location. At least that's how it currently works. Maybe this will change if they do make "the algorithm" public :p
